primer/brand
# Add width and letter spacing configurability to Heading

## Summary

Part of https://github.com/github/galaxy.github.com/issues/118

Adds new `width` prop to `Heading` to allow granular control over variable font capabilities.

Widths based on presets defined in the [Mona Sans style table](https://github.com/github/mona-sans). 

Also added `letterSpacing` to allow granular adjustment over `letter-spacing` styles, which is necessary when updating the `width`

## Steps to test:



1. Check out the [docs page](https://primer-11843cfdec-26139705.drafts.github.io/components/Heading#width)
1. Check out the new stories
1. Review proposed API 

## Supporting resources (related issues, external links, etc):

- [Mona Sans](https://github.com/github/mona-sans)
- [Galaxy issue](https://github.com/github/galaxy.github.com/issues/118)

## Contributor checklist:


## Reviewer checklist:


## Screenshots:
![width](https://user-images.githubusercontent.com/13340707/218765724-3bde8de7-2fcc-40b1-8a76-16d7c49cec11.png)
![letter-spacing](https://user-images.githubusercontent.com/13340707/218765734-d4971e0b-5120-4d94-9cad-1f130a3205bc.png)
